.github/workflows/hub.yml

```diff
@@ -50,6 +53,6 @@ jobs:
           # cache-to: type=local,dest=/tmp/.buildx-cache
           tags: |
             akhilrex/hammond:latest
-            akhilrex/hammond:1.0.0
+            akhilrex/hammond:${{ steps.get_tag.outputs.TAG }}
```

Comment:
These might need to be changed for the fork. But I think it's good for now. We'll change it if we notice any issues.